lib/elixir_internal_certification_web/helpers/data_helper.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ElixirInternalCertificationWeb.Helpers.KeywordHelper do
+  alias ElixirInternalCertification.Account.Schemas.User
+  alias ElixirInternalCertification.Keyword.{KeywordLookups, Keywords}
+
+  def process_upload(%User{} = current_user, path) do
+    with {:ok, keywords} <- Keywords.parse_csv!(path),
+         {:ok, {_, records}} <- Keywords.create_keywords(current_user, keywords),
+         scheduled_keyword_lookups <- Enum.map(records, &KeywordLookups.schedule_keyword_lookup/1) do
+      {:ok, {path, records, scheduled_keyword_lookups}}
+    else
+      {:error, reason} -> {:error, reason}
```

Comment:
```suggestion
     error ->
        error
```
I am fine with either approach but this will just send the error straight back in the same format (which is what you are doing anyway). I guess one benefit of keeping it explicit like you have is if you add a function in there somewhere and it returns a different response to {:error, reason} you may get compile time help you are not catching it so fine with either 👍